src/components/SingleSelect/SingleSelect.spec.tsx

```diff
@@ -278,6 +278,24 @@ describe('SingleSelect', () => {
 				});
 			});
 		});
+
+		describe('has a title', () => {
+			it('should have a `Title`', () => {
+				const title = 'Foo';
+
+				const wrapper = shallow(
+					<SingleSelect Title={title}>
+						<Placeholder>select one</Placeholder>
+						<Option>option a</Option>
+						<Option>option b</Option>
+						<Option>option c</Option>
+					</SingleSelect>
+				);
+
+				const titleWrapper = wrapper.find('.select-title');
+				assert.equal(titleWrapper.text(), `${title}:`);
```

Comment:
Just a preference FYI, CXP has recently stopped using the assert library for new tests and is using expect instead.

Reply:
@yasskin Makes sense.  efe0af5380f68eeeac7539442da15c63833e2407